### PR TITLE
New package: neofetch-1.5

### PR DESCRIPTION
**Pull request by konimex**
Let's try this again.

### CODE REVIEW

File: srcpkgs/neofetch/template [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+depends="bash xprop ImageMagick w3m-img"
+short_desc="Simple system information script"
+maintainer="Herdiansyah <herdiansyah@opmbx.org>"
+homepage="https://github.com/dylanaraps/neofetch"
+license="MIT"
+distfiles="${homepage}/archive/${version}/${pkgname}-${version}.tar.gz"
+checksum=b0bda1ac5c3fa4357c6c2157767bcbf08d0e231e3d5e15c4408490f1c3175756
+
+post_install() {
+vlicense LICENSE.md
```
Comment (Vaelatern) [inline srcpkgs/neofetch/template]: Increase indentation please.

Comment (konimex) [inline srcpkgs/neofetch/template]: On 03/14/2016 12:29 AM, Toyam Cox wrote:
> Increase indentation please.

Done, for some reason xlint said that it wasn't normal to have a tab
indentation, though.

Comment (konimex) [inline srcpkgs/neofetch/template]: Something's wrong with my vim config, I'll look into that, fixed that problem
with xlint.

Comment (Vaelatern) [inline srcpkgs/neofetch/template]: It's not normal to use spaces instead of tabs. You probably have `:set expandtab`

**Comment by konimex** [comment]
Goddammit. My fault. I'll fix it for a bit.

**Pull request merged**